_i18n/es.yml

```diff
@@ -163,6 +163,27 @@ help:
 help_page:
   p_1: Explore los temas comunes
   title: ¿Cómo podemos ayudar?
+  categories:
+    - title: Empiece con login.gov
+      description: Cree su cuenta. Conozca las opciones de autenticación y más características de la cuenta.
+      url: /help/get-started/overview/
+      image: /assets/img/help/get-started.svg
+    - title: ¿Tiene problemas para iniciar sesión?
+      description: ¿Olvidó su contraseña? ¿Se bloqueó su cuenta? Le ayudaremos a solucionar los problemas de acceso.
+      url: /help/trouble-signing-in/overview/
+      image: /assets/img/help/trouble-signing-in.svg
+    - title: Administrar su cuenta
+      description: Cambia la configuración de la cuenta, incluida la contraseña, el teléfono, el correo electrónico y más.
+      url: /help/manage-your-account/overview/
+      image: /assets/img/help/manage-your-account.svg
+    - title: Asistencia de agencias específicas
+      description: Obtenga ayuda con Trusted Traveler Programs (TTP), SAM.gov.
```

Comment:
There's a comma between "(TTP)" and "SAM.gov." It's missing the Spanish equivalent of "and" since we removed "USAJOBS"

We should consider updating to "(TTP) y SAM.gov"

Reply:
I thought about this...but the translator omitted the y. Wasn't sure if there was a reason for this.